v0/destinations/hs/HSTransform-v2.js

```diff
@@ -50,52 +52,66 @@ const processIdentify = async (message, destination, propertyMap) => {
   const { Config } = destination;
   const traits = getFieldValueFromMessage(message, "traits");
   const mappedToDestination = get(message, MappedToDestinationKey);
+  const checkLookup = get(message, "context.hubspotOperation");
```

Comment:
please rename this to something relevant like hubspotOp? checkLookup doesn't describe the value it is holding

Reply:
done